docs/types/array.md

```diff
@@ -124,28 +154,15 @@ Returns a new array with only the elements that returned
 ["hello", 0, 1, 2].filter(f(x){type(x) == "NUMBER"}) # [0, 1, 2]
 ```
 
-### json()
```

Comment:
I think this has been removed by mistake

Reply:
I removed json() from array.md because it is correctly documented in string.md as string function. I think this was a typo as json() should not be duplicated in array.md where it does not apply.